packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -78,15 +81,24 @@ export class RedirectsMiddleware extends MiddlewareBase {
     });
 
     const createResponse = async () => {
+      const response = res || NextResponse.next();
+
       if (this.config.disabled && this.config.disabled(req, res || NextResponse.next())) {
         debug.redirects('skipped (redirects middleware is disabled)');
-        return res || NextResponse.next();
+        return response;
       }
 
       if (this.isPreview(req) || this.excludeRoute(pathname)) {
         debug.redirects('skipped (%s)', this.isPreview(req) ? 'preview' : 'route excluded');
 
-        return res || NextResponse.next();
+        return response;
+      }
+
+      // Skip prefetch requests
+      if (this.isPrefetch(req)) {
```

Comment:
Let's add some explanation on the reason of skipping prefetch requests, for better understanding if other people will need to figure out the scenario